src/Engines/Markdown.js

```diff
@@ -79,9 +79,8 @@ class Markdown extends TemplateEngine {
           return str;
         };
       } else {
-        return function() {
-          // throw away data if preTemplateEngine is falsy
```

Comment:
Why did you drop the comment?

Reply:
From what I can tell the comment is not relevant anymore

Reply:
As @MadeByMike mentioned, the comment says, "throw away data", but we're not throwing away the data any more.